src/strand-grid-item/strand-grid-item.html

```diff
@@ -34,7 +34,9 @@
 			</div>
 		</div>
 		<template is="dom-if" if="{{!model}}">
-			<strand-spinner id="loader" radius="12" fill-color="#333333"></strand-spinner>
+			<div class="_mm_item">
```

Comment:
we actually probably dont want this cause it will confuse a bunch of existing users